### PR TITLE
[Bugfix:Submission] Notebook unnecessary UI elements

### LINKED ISSUE #8338: Anon Ids missing for recently added students
**Describe the bug**
If we add a student to the course after one or more gradeables have been created, the student will not have an anon id for that gradeable.  This causes weird bugs...

* If the newly added student tries to access the gradeable [ notebook gradeable does not render, irrelevant errors about teams, etc. ]

* When a grader goes to view the grading for that assignment, the newly added student is not listed in the list of all students in the course.

The anon_id table is missing any anon id for the newly added students for these existing gradeables.

### PR DESCRIPTION
**Pull request by Eceptonsu**
### What is the current behavior?
Closes #8338 
Currently there are submission mode and bulk upload processing progress box on notebook (and other non-bulk) gradeables for full access graders, which is unnecessary.

### What is the new behavior?
Remove the processing progress box on notebook (and other non-bulk) gradeables, and the submit for student option works for notebook now.

### CODE REVIEW

File: site/app/templates/submission/homework/SubmitBox.twig [hunk 1]
```diff
@@ -64,138 +64,140 @@
...
         {% endif %}
+        {# /Admin submission type selector #}
+
+        {% if is_vcs %}
+            {% if vcs_host_type == 0 or vcs_host_type == 1 %}
+                <h3>To access your Repository:</h3>
+                <span><em>Note: There may be a delay before your repository is prepared, please refer to assignment instructions.</em></span><br />
+                <samp>git  clone  {{ repository_path }}  SPECIFY_TARGET_DIRECTORY</samp>
+                {# Below will get uncommented once SAML is active #}
+                {# <p>You will need to use an Authentication Token.</p> #}
```
Comment (Viyerelu23333) [inline site/app/templates/submission/homework/SubmitBox.twig:167]: Should this line be uncommented now?

**Review by Viyerelu23333 — APPROVED** [review]
Looks good to me and worked on my end.

**Review by shailpatels — CHANGES_REQUESTED** [review]

File: site/ts/grader-submission.ts [hunk 2]
```diff
@@ -21,7 +21,9 @@ function init() {
     document.getElementsByName('submission-type')
         .forEach(radio_btn => radio_btn.addEventListener('click', changeSubmissionMode));
 
-    warning_banner!.textContent = '';
+    if (warning_banner) {
```
[the remark this answers is not in the document]
Comment (Eceptonsu) [reply site/ts/grader-submission.ts:24]: I think these checks remove the console error for notebooks

File: site/app/templates/submission/homework/SubmitBox.twig [hunk 3]
```diff
@@ -356,7 +356,9 @@
 
 {% if core.getUser().accessFullGrading() %}
     {% include 'submission/homework/PreviousSubmissionForm.twig' %}
-    {% include 'submission/homework/BulkUploadProgressBox.twig' %}
+    {% not is_vcs and not is_notebook %}
```
Comment (shailpatels) [inline site/app/templates/submission/homework/SubmitBox.twig:359]: I think you forgot the `if` here, twig is complaining this isn't a block when you visit a gradeable view grade page

```

Server Error
Oh no! Something irrecoverable has happened...

app\exceptions\OutputException (Code: 0) thrown in /usr/local/submitty/site/app/libraries/Output.php (Line 371) by:
            throw new OutputException("{$e->getMessage()} in {$e->getFile()}:{$e->getLine()}");


Message:
A block must start with a tag name. in /usr/local/submitty/site/app/templates/submission/homework/SubmitBox.twig:359

Stack Trace:
#0 /usr/local/submitty/site/app/views/submission/HomeworkView.php(457): app\libraries\Output->renderTwigTemplate()
#1 /usr/local/submitty/site/app/views/submission/HomeworkView.php(64): app\views\submission\HomeworkView->renderSubmitBox()
#2 unknown file(unknown line): app\views\submission\HomeworkView->showGradeable()
#3 /usr/local/submitty/site/app/libraries/Output.php(233): call_user_func_array()
#4 /usr/local/submitty/site/app/libraries/Output.php(202): app\libraries\Output->renderTemplate()
#5 /usr/local/submitty/site/app/controllers/student/SubmissionController.php(194): app\libraries\Output->renderOutput()
#6 unknown file(unknown line): app\controllers\student\SubmissionController->showHomeworkPage()
#7 /usr/local/submitty/site/app/libraries/routers/WebRouter.php(254): call_user_func_array()
#8 /usr/local/submitty/site/app/libraries/routers/WebRouter.php(232): app\libraries\routers\WebRouter->run()
#9 /usr/local/submitty/site/public/index.php(131): app\libraries\routers\WebRouter::getWebResponse()

                      ,--.    ,--.
                     (( O))--(( O))
                   ,'_`--'____`--'_`.
                  _:  ____________  :_      _____
                 | | ||::::::::::|| | |        \ \
                 | | ||::::::::::|| | |         \
                 | | ||::::::::::|| | |
                 |_| |/__________\| |_|
                   |________________|
                __..-'            `-..__
             .-| : .----------------. : |-.
           ,\ || | |\______________/| | || /.
          /`.\:| | ||  __  __  __  || | |;/,'\
         :`-._\;.| || '--''--''--' || |,:/_.-':
         |    :  | || .----------. || |  :    |
         |    |  | || '----SSt---' || |  |    |
         |    |  | ||   _   _   _  || |  |    |
         :,--.;  | ||  (_) (_) (_) || |  :,--.;
         (`-'|)  | ||______________|| |  (|`-')
          `--'   | |/______________\| |   `--'
                 |____________________|
                  `.________________,'
                   (_______)(_______)
                   (_______)(_______)
                   (_______)(_______)
                   (_______)(_______)
                  |        ||        |
                  '--------''--------'
    

````

**Review by shailpatels — CHANGES_REQUESTED** [review]

File: site/ts/grader-submission.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (shailpatels) [reply site/ts/grader-submission.ts:24]: Ok I see that notebook gradeables have the grader submission option of `submit for student` but not bulk submit. Could you add a comment somewhere in that file around the if checks explaining that notebook gradeables don't have the same set of UI elements as other gradeables for instructors and that's why those null checks exist

[the remark this answers is not in the document]
Comment (Eceptonsu) [reply site/ts/grader-submission.ts:24]: Will do.

File: site/app/templates/submission/homework/SubmitBox.twig (hunk 3 above)
Comment (Eceptonsu) [reply site/app/templates/submission/homework/SubmitBox.twig]: Sorry forgot to push the the latest commit

**Review by shailpatels — APPROVED** [review]
Changes look good and fixes the issue!

**Pull request merged**